api/v2/checluster_types.go

```diff
@@ -366,15 +366,16 @@ type TrustedCerts struct {
 
 // Configuration settings related to the workspaces persistent storage.
 type WorkspaceStorage struct {
-	// PVC settings.
+	// PVC settings when using the "per-user" PVC strategy.
```

Comment:
I don't think that's only for `per-user` (i.e. the `pvcStrategy` is configured in this same struct).

Reply:
And I would make it clear that those settings are about the Development environments PVs.

Reply:
Ok sorry my bad, the comment above is for the `perUserStrategyPvcConfig`. The diff confused me. Don't consider my comments above.